django/db/models/expressions.py

```diff
@@ -1072,11 +1080,14 @@ def as_oracle(self, compiler, connection, template=None, **extra_context):
         # requires a left hand side (column) to compare against.
         sql, params = self.as_sql(compiler, connection, template, **extra_context)
         sql = 'CASE WHEN {} THEN 1 ELSE 0 END'.format(sql)
+        if self._used_in_where:
+            sql = '{} = True'.format(sql)
```

Comment:
You should use `1` instead of `True`, sorry for misleading. If you will use `True` as a param than it will be automatically converted into `1`.